app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+def copy_file(command: str) -> None:
+    _, copy_from, copy_to = command.split()
```

Comment:
Do not omit the command name, you should add a check that the command is actually `cp` before performing any other actions